test/jdk/javax/swing/ToolTipManager/bug5078214.java

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright (c) 2004, 2023, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+/*
+ * @test
+ * @bug 5078214
+ * @summary ToolTip is shown partially when the application is near the bottom of screen.
+ * @library ../regtesthelpers /test/lib
+ * @build JRobot
+ * @build jtreg.SkippedException
```

Comment:
```suggestion
 * @library /test/lib
 * @build jtreg.SkippedException
```
JRobot is not required for this test, it can be replaced with normal Robot calls and jtreg header can be modified as above.

Reply:
As per convention it is better to place the `@key headful` at the start after `@test or @bug`.

Reply:
@alisenchung You missed removing `../regtesthelpers` from @library tag  :)
Also please move the @key tag as suggested above.